testsuite/cluster-test/src/health/debug_interface_log_tail.rs

```diff
@@ -42,7 +42,8 @@ impl DebugPortLogThread {
         for instance in cluster.all_instances() {
             let (started_sender, started_receiver) = mpsc::channel();
             started_receivers.push(started_receiver);
-            let client = NodeDebugClient::new(instance.ip(), 6191);
+            let client =
+                NodeDebugClient::new(instance.ip(), instance.debug_interface_port().unwrap());
```

Comment:
Can you replace `.unwrap` with `.expect("Some sensible message about debug interface port required for debug log thread")`